### PR TITLE
Revised documentation

### PR DESCRIPTION
**Pull request by hamid-nazari**
I've been studying documentation for past few days and came across some minor issues that I felt would be a pity to let hem slip. Such a great work deserves perfection for all that matters.

Hope these are seen relevant and get pulled in.

### CODE REVIEW

File: docs/index.adoc [hunk 1]
```diff
@@ -8060,7 +8060,7 @@ class ListFiles : Runnable {
 ----
 
 === Subcommand Aliases
-Commands may optionally define an `aliases` attribute to provide alternate names for commands that will be recognized by the parser. Aliases are displayed in the default help output. For example:
+Commands may optionally define an `aliases` attribute to provide alternate names for subcommands that will be recognized by the parser. Aliases are displayed in the default help output. For example:
```
Comment (remkop) [inline docs/index.adoc:8063]: Reading this again, I realized that maybe this "for commands" or "for subcommands" can be confusing. 
How about leaving it out altogether? So we get:

> ... provide alternate names that will be recognized by the parser.

File: docs/index.html [hunk 2]
```diff
@@ -12344,7 +12344,7 @@ <h3 id="parentcommand-annotation"><a class="anchor" href="#parentcommand-annotat
 <div class="sect2">
 <h3 id="_subcommand_aliases"><a class="anchor" href="#_subcommand_aliases"></a>17.8. Subcommand Aliases</h3>
 <div class="paragraph">
-<p>Commands may optionally define an <code>aliases</code> attribute to provide alternate names for commands that will be recognized by the parser. Aliases are displayed in the default help output. For example:</p>
+<p>Commands may optionally define an <code>aliases</code> attribute to provide alternate names for subcommands that will be recognized by the parser. Aliases are displayed in the default help output. For example:</p>
```
Comment (remkop) [inline docs/index.html:12347]: Same as in the .adoc file:

> provide alternate names that will be recognized by the parser.

File: docs/man/3.x/index.html [hunk 3]
```diff
@@ -3815,7 +3815,7 @@ <h3 id="_registering_subcommands_declaratively">13.2. Registering Subcommands De
 <div class="sect2">
 <h3 id="_subcommand_aliases">13.3. Subcommand Aliases</h3>
 <div class="paragraph">
-<p>Commands may optionally define an <code>aliases</code> attribute to provide alternate names for commands that will be recognized by the parser. Aliases are displayed in the default help output. For example:</p>
+<p>Commands may optionally define an <code>aliases</code> attribute to provide alternate names for subcommands that will be recognized by the parser. Aliases are displayed in the default help output. For example:</p>
```
Comment (remkop) [inline docs/man/3.x/index.html:3818]: Let's leave out the "for commands" and "for subcommands" altogether:

> ... provide alternate names that will be recognized by the parser.

**Review by remkop — APPROVED** [review]
Wow this is great! Thank you so much!

I found one small improvement in how to phrase the explanation for aliases.
Can you take a look?
If you don't have time to fix that, let me know, I can do it later.

Thank you again! 🙏

File: docs/index.adoc (hunk 1 above)
Comment (hamid-nazari) [reply docs/index.adoc:8063]: I agree, it's better this way

File: docs/index.html (hunk 2 above)
Comment (hamid-nazari) [reply docs/index.html:12347]: 👍

File: docs/man/3.x/index.html (hunk 3 above)
Comment (hamid-nazari) [reply docs/man/3.x/index.html:3818]: 👍

**Comment by remkop** [comment]
@hamid-nazari I saw your feedback, thank you. 🙏 
Will you be able to update your PR, or do you want me to make the changes after merging it?

**Comment by hamid-nazari** [comment]
Sorry @remkop,
I got caught up last night and forgot to push my changes

**Comment by remkop** [comment]
Merged. 
Thank you for the contribution!

**Pull request merged**